test/test_Font.py

```diff
@@ -0,0 +1,84 @@
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301 USA
+# This file is part of pyOCCT which provides Python bindings to the OpenCASCADE
+# geometry kernel.
+#
+# Copyright (C) 2016-2018  Laughlin Research, LLC (info@laughlinresearch.com)
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301 USA
+
+FONT_CACHE = {}
+
+def test_font_create_shape():
+    from OCCT.Font import (
+        Font_FontMgr, Font_BRepTextBuilder, Font_FontAspect, Font_BRepFont
+    )
+
+    from OCCT.TCollection import TCollection_AsciiString
+    from OCCT.NCollection import NCollection_String
+    from OCCT.TColStd import TColStd_SequenceOfHAsciiString
+    from OCCT.Graphic3d import Graphic3d_HTA_LEFT, Graphic3d_VTA_BOTTOM
+    from OCCT.gp import gp_Ax3
+
+    # Get a font (os dependent)
+    font_names = TColStd_SequenceOfHAsciiString()
+    font_mgr = Font_FontMgr.GetInstance_()
+    font_mgr.GetAvailableFontsNames(font_names)
+    assert font_names.Size()
+
+    font_family = TCollection_AsciiString(font_names.First().ToCString())
+
+    font_style = Font_FontAspect.Font_FA_Regular
+
+    # Create the font
+    font = Font_BRepFont()
+    assert font.FindAndInit(font_family, font_style, 12.0)
+
+    # The font should be saved somewhere after creating it!
+    key = (font_family, font_style, 12.0)
+    FONT_CACHE[key] = font
+
+    # Create builder
+    builder = Font_BRepTextBuilder()
+
+    # Parameters
+    text = NCollection_String("pyOCCT".encode("utf-8"))
+    pos = gp_Ax3()
+    halign = Graphic3d_HTA_LEFT
+    valign = Graphic3d_VTA_BOTTOM
+    topods_shape = builder.Perform(font, text, pos, halign, valign)
+
+    assert not topods_shape.IsNull()
+
+
+
+
```

Comment:
can you write the unit tests in the format similar to the existing ones in test folder like https://github.com/LaughlinResearch/pyOCCT/blob/master/test/test_Geom.py